### ISSUE TITLE
Connection failure on fresh install

### ISSUE THREAD

**Issue by radriaanse** [jellyfin/jellyfin#3096]
**Describe the bug**

Initially did an upgrade from 10.5.4, but I screwed some things up in my libraries while playing around so decided to do a clean install. Removed all packages and data paths (excluding library data) and installed 10.5.5 from the AUR.
Connecting via an nginx reverse proxy that was already in place, I get the first use wizard, but the moment I click `next` the page hangs on a spinner.
The console log shows:
```
uncaught exception: Object
app is hidden apphost.js:205:21
triggering app resume event apphost.js:197:21
error opening web socket: Error: Cannot open web socket without access token. apiclientcore.js:363:21
```

And the server log:
```
May 11 23:54:45 myserver dotnet[18758]: [23:54:45] [INF] [15] Jellyfin.Api.Auth.CustomAuthenticationHandler: CustomAuthentication was not authenticated. Failure message: Access token is required.
```
Though that it might have to do with some network setting, Nginx terminates tls and serves the app on a subdomain `proxy_pass`'ed to 127.0.0.1:8096; so I tried connecting to the IP directly.
This works a bit better, get the wizard again but still get
```
May 12 00:08:41 myserver dotnet[18758]: [00:08:41] [INF] [12] Jellyfin.Api.Auth.CustomAuthenticationHandler: CustomAuthentication was not authenticated. Failure message: Access token is required.
```
In the server log, but can however click through. Created a new user, skipped library setup, all default settings.
Can login afterwards via the IP:
```
May 12 00:10:48 myserver dotnet[18758]: [00:10:48] [INF] [12] Emby.Server.Implementations.Library.UserManager: Authentication request for testuser has succeeded.
May 12 00:10:48 myserver dotnet[18758]: [00:10:48] [INF] [12] Emby.Server.Implementations.Session.SessionManager: Creating new access token for user 7f20a453-ecfc-4bdd-b304-6e6b8e6e9be2
May 12 00:10:49 myserver dotnet[18758]: [00:10:49] [INF] [4] App: WS http://203.0.113.1:8096/socket?api_key=48d177e707954e16a0ea2296744555c1&deviceId=TW96aWxsYS81LjAgKFgxMTsgTGludXggeDg2XzY0OyBydjo3NC4wKSBHZWNrby8yMDEwMDEwMSBGaXJlZm94Lzc0LjB8MTU4OTIzNDE0NTkzMA11. UserAgent: Mozilla/5.0 (X11; Linux x86_64; rv:74.0) Gecko/20100101 Firefox/74.0
May 12 00:11:30 myserver dotnet[18758]: [00:11:30] [INF] [4] Emby.Server.Implementations.Session.SessionManager: Logging out access token 48d177e707954e16a0ea2296744555c1
```

but once I switch back to the reverse proxy it fails with:

> Connection Failure
We're unable to connect to the selected server right now. Please ensure it is running and try again.

And the console log shows the websocket error again.
Tried to restore the networking settings as they were before, but that doesn't seem change anything (public http/https -> 80/443, secure connection -> reverse proxy, bind -> 127.0.0.1)

**System**
 - OS: Arch (Linux version 5.6.11-arch1-1)
 - Virtualization: KVM
 - Clients: Web
 - Browser: Firefox 74.0 & Chromium 80.0.3987.149
 - Jellyfin Version: 10.5.5
 - Playback: N/A
 - Installed Plugins: N/A
 - Reverse Proxy: Nginx 1.18.0
 - Base URL: none (/), subdomain
 - Networking: N/A
 - Storage: local

**To Reproduce**
Install 10.5.5
Walk through wizard
Setup reverse proxy, i.e. Nginx:

```
server {
        server_name     jellyfin.myserver.tld;
        listen          203.0.113.1:80;

        location / {
                return 301 https://jellyfin.myserver.tld$request_uri;
        }
}

server {
        server_name             jellyfin.myserver.tld;
        listen                  203.0.113.1:443 ssl http2;
        ssl_certificate         /etc/ssl/private/myserver.crt;
        ssl_certificate_key     /etc/ssl/private/myserver.key;
        ssl_trusted_certificate /etc/ssl/private/myserver.crt;

        location / {
                proxy_pass http://127.0.0.1:8096;

                proxy_set_header Host $host;
                proxy_set_header X-Real-IP $remote_addr;
                proxy_set_header X-Forwarded-for $proxy_add_x_forwarded_for;
                proxy_set_header X-Forwarded-Proto $scheme;
                proxy_set_header X-Forwarded-Protocol $scheme;
                proxy_set_header X-Forwarded-Host $http_host;

                proxy_http_version 1.1;
                proxy_set_header Upgrade $http_upgrade;
                proxy_set_header Connection "upgrade";

                proxy_buffering off;
                proxy_redirect off;
        }
}
```
Try to login with existing user.

**Expected behavior**
Expected this to work, it did before a clean install actually on the same version and same setup.

**Logs**
Relevant logs included above, debug doesn't give any additional info at all.

**Screenshots**
Included as text above

Seems similar to #3091 but I do not get the same exception in log.

**Comment by motion-x86** [comment]
Same issue here running the jellyfin/jellyfin image for arm sha256:26598d1fd9516308ba36b771d0d9271d17b8eeb51276684ebd2b6737be41e199

can't go past the very first screen of the wizard "Preferred display language:"

jellyfin sits behind nginx, nginx terminates tls and does a proxypass to the jellyfin service:

`server {
    listen 443 ssl;

    server_name "jellyfin.myhomedomain.net";

    include /etc/nginx/extra/ssl.conf;

    set $jellyfin_server jellyfin:8096;

    resolver 127.0.0.11 valid=30s ipv6=off;

    location / {
        proxy_set_header Range $http_range;
        proxy_set_header If-Range $http_if_range;
        proxy_set_header X-Forwarded-Proto $scheme;
#        proxy_set_header X-Forwarded-Protocol $scheme;
        proxy_set_header X-Forwarded-Host $http_host;

        # allow ws connections by setting the two headers below
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection "Upgrade";
        proxy_http_version 1.1;
        proxy_pass http://$jellyfin_server$request_uri;

        proxy_buffering off;
    }

    location /socket {
       # Proxy Jellyfin Websockets traffic
       proxy_pass http://$jellyfin_server$request_uri;

       proxy_http_version 1.1;
       proxy_set_header Upgrade $http_upgrade;
       proxy_set_header Connection "upgrade";

       proxy_set_header X-Forwarded-Proto $scheme;
       #proxy_set_header X-Forwarded-Protocol $scheme;
       proxy_set_header X-Forwarded-Host $http_host;
   }
}`

docker setup:
`version: "3.7"
services:
  jellyfin:
    image: jellyfin/jellyfin
    user: 1000:1000

    deploy:
      replicas: 1
      placement:
        constraints:
          - "node.labels.samba==true"
          - "node.labels.arch==arm"

    volumes:
      - /conf/jellyfin:/config
      - /mnt/storage/jellyfin/cache:/cache
      - /mnt/storage/media:/media

    networks:
      - frontend

networks:
  frontend:
    name: frontend
    external: true`

container logs:

`[09:12:51] [INF] [16] TaskManager: IntervalTrigger fired for task: Clean Log Directory,
[09:12:51] [INF] [16] TaskManager: Queueing task DeleteLogFileTask,
[09:12:51] [INF] [23] TaskManager: Executing Clean Log Directory,
[09:12:51] [INF] [23] TaskManager: Clean Log Directory Completed after 0 minute(s) and 0 seconds,
[09:12:51] [INF] [23] TaskManager: ExecuteQueuedTasks,
[09:12:51] [INF] [14] TaskManager: IntervalTrigger fired for task: Refresh Channels,
[09:12:51] [INF] [14] TaskManager: Queueing task RefreshChannelsScheduledTask,
[09:12:51] [INF] [4] TaskManager: Executing Refresh Channels,
[09:12:51] [INF] [4] TaskManager: Refresh Channels Completed after 0 minute(s) and 0 seconds,
[09:12:51] [INF] [4] TaskManager: ExecuteQueuedTasks,
[09:12:52] [INF] [14] TaskManager: IntervalTrigger fired for task: Refresh Users,
[09:12:52] [INF] [14] TaskManager: Queueing task RefreshUsersMetadata,
[09:12:52] [INF] [23] TaskManager: Executing Refresh Users,
[09:12:52] [INF] [23] TaskManager: Refresh Users Completed after 0 minute(s) and 0 seconds,
[09:12:52] [INF] [23] TaskManager: ExecuteQueuedTasks,
[09:12:53] [INF] [4] TaskManager: IntervalTrigger fired for task: Clean Cache Directory,
[09:12:53] [INF] [4] TaskManager: Queueing task DeleteCacheFileTask,
[09:12:53] [INF] [16] TaskManager: Executing Clean Cache Directory,
[09:12:53] [INF] [16] TaskManager: Clean Cache Directory Completed after 0 minute(s) and 0 seconds,
[09:12:53] [INF] [16] TaskManager: ExecuteQueuedTasks,
[09:12:58] [INF] [4] TaskManager: IntervalTrigger fired for task: Scan Media Library,
[09:12:58] [INF] [4] TaskManager: Queueing task RefreshMediaLibraryTask,
[09:12:58] [INF] [14] TaskManager: Executing Scan Media Library,
[09:12:58] [INF] [14] LibraryManager: Validating media library,
[09:12:58] [INF] [14] TaskManager: Scan Media Library Completed after 0 minute(s) and 0 seconds,
[09:12:58] [INF] [14] TaskManager: ExecuteQueuedTasks,
[10:12:58] [INF] [4] TaskManager: IntervalTrigger fired for task: Refresh Guide,
[10:12:58] [INF] [4] TaskManager: Queueing task RefreshChannelsScheduledTask,
[10:12:58] [INF] [4] TaskManager: Executing Refresh Guide,
[10:13:01] [INF] [17] LiveTvManager: Refreshing guide with 7 days of guide data,
[10:13:01] [INF] [17] TaskManager: Refresh Guide Completed after 0 minute(s) and 3 seconds,
[10:13:01] [INF] [17] TaskManager: ExecuteQueuedTasks,
[21:12:58] [INF] [18] TaskManager: IntervalTrigger fired for task: Scan Media Library,
[21:12:58] [INF] [18] TaskManager: Queueing task RefreshMediaLibraryTask,
[21:12:58] [INF] [22] TaskManager: Executing Scan Media Library,
[21:12:58] [INF] [22] LibraryManager: Validating media library,
[21:12:58] [INF] [22] TaskManager: Scan Media Library Completed after 0 minute(s) and 0 seconds,
[21:12:58] [INF] [22] TaskManager: ExecuteQueuedTasks,
[01:59:59] [INF] [4] TaskManager: DailyTrigger fired for task: Extract Chapter Images,
[01:59:59] [INF] [4] TaskManager: Queueing task ChapterImagesTask,
[01:59:59] [INF] [4] TaskManager: Executing Extract Chapter Images,
[02:00:00] [INF] [4] TaskManager: Extract Chapter Images Completed after 0 minute(s) and 0 seconds,
[02:00:00] [INF] [4] TaskManager: ExecuteQueuedTasks,
[02:00:01] [INF] [9] TaskManager: Daily trigger for Extract Chapter Images set to fire at 06/11/2020 02:00, which is 23:59:58.9991182 from now.,
[08:36:51] [INF] [6] TaskManager: IntervalTrigger fired for task: Update Plugins,
[08:36:51] [INF] [6] TaskManager: Queueing task PluginUpdateTask,
[08:36:51] [INF] [16] TaskManager: Executing Update Plugins,
[08:36:52] [INF] [19] TaskManager: Update Plugins Completed after 0 minute(s) and 1 seconds,
[08:36:52] [INF] [19] TaskManager: ExecuteQueuedTasks,
[09:12:51] [INF] [16] TaskManager: IntervalTrigger fired for task: Download missing subtitles,
[09:12:51] [INF] [16] TaskManager: Queueing task SubtitleScheduledTask,
[09:12:51] [INF] [21] TaskManager: Executing Download missing subtitles,
[09:12:51] [INF] [21] TaskManager: Download missing subtitles Completed after 0 minute(s) and 0 seconds,
[09:12:51] [INF] [21] TaskManager: ExecuteQueuedTasks,
[09:12:51] [INF] [21] TaskManager: IntervalTrigger fired for task: Clean Log Directory,
[09:12:51] [INF] [21] TaskManager: Queueing task DeleteLogFileTask,
[09:12:51] [INF] [21] TaskManager: Executing Clean Log Directory,
[09:12:51] [INF] [21] TaskManager: Clean Log Directory Completed after 0 minute(s) and 0 seconds,
[09:12:51] [INF] [21] TaskManager: ExecuteQueuedTasks,
[09:12:51] [INF] [10] TaskManager: IntervalTrigger fired for task: Refresh Channels,
[09:12:51] [INF] [10] TaskManager: Queueing task RefreshChannelsScheduledTask,
[09:12:51] [INF] [10] TaskManager: Executing Refresh Channels,
[09:12:51] [INF] [10] TaskManager: Refresh Channels Completed after 0 minute(s) and 0 seconds,
[09:12:51] [INF] [10] TaskManager: ExecuteQueuedTasks,
[09:12:52] [INF] [19] TaskManager: IntervalTrigger fired for task: Refresh Users,
[09:12:52] [INF] [19] TaskManager: Queueing task RefreshUsersMetadata,
[09:12:52] [INF] [19] TaskManager: Executing Refresh Users,
[09:12:52] [INF] [19] TaskManager: Refresh Users Completed after 0 minute(s) and 0 seconds,
[09:12:52] [INF] [19] TaskManager: ExecuteQueuedTasks,
[09:12:53] [INF] [19] TaskManager: IntervalTrigger fired for task: Clean Cache Directory,
[09:12:53] [INF] [19] TaskManager: Queueing task DeleteCacheFileTask,
[09:12:53] [INF] [19] TaskManager: Executing Clean Cache Directory,
[09:12:53] [INF] [19] TaskManager: Clean Cache Directory Completed after 0 minute(s) and 0 seconds,
[09:12:53] [INF] [19] TaskManager: ExecuteQueuedTasks,
[09:12:58] [INF] [21] TaskManager: IntervalTrigger fired for task: Scan Media Library,
[09:12:58] [INF] [21] TaskManager: Queueing task RefreshMediaLibraryTask,
[09:12:58] [INF] [21] TaskManager: Executing Scan Media Library,
[09:12:58] [INF] [21] LibraryManager: Validating media library,
[09:12:58] [INF] [21] TaskManager: Scan Media Library Completed after 0 minute(s) and 0 seconds,
[09:12:58] [INF] [21] TaskManager: ExecuteQueuedTasks,
[10:12:58] [INF] [21] TaskManager: IntervalTrigger fired for task: Refresh Guide,
[10:12:58] [INF] [21] TaskManager: Queueing task RefreshChannelsScheduledTask,
[10:12:58] [INF] [19] TaskManager: Executing Refresh Guide,
[10:13:01] [INF] [4] LiveTvManager: Refreshing guide with 7 days of guide data,
[10:13:01] [INF] [4] TaskManager: Refresh Guide Completed after 0 minute(s) and 3 seconds,
[10:13:01] [INF] [4] TaskManager: ExecuteQueuedTasks,
[21:12:58] [INF] [23] TaskManager: IntervalTrigger fired for task: Scan Media Library,
[21:12:58] [INF] [23] TaskManager: Queueing task RefreshMediaLibraryTask,
[21:12:58] [INF] [25] TaskManager: Executing Scan Media Library,
[21:12:58] [INF] [25] LibraryManager: Validating media library,
[21:12:58] [INF] [25] TaskManager: Scan Media Library Completed after 0 minute(s) and 0 seconds,
[21:12:58] [INF] [25] TaskManager: ExecuteQueuedTasks,
[01:59:59] [INF] [15] TaskManager: DailyTrigger fired for task: Extract Chapter Images,
[01:59:59] [INF] [15] TaskManager: Queueing task ChapterImagesTask,
[01:59:59] [INF] [27] TaskManager: Executing Extract Chapter Images,
[02:00:00] [INF] [27] TaskManager: Extract Chapter Images Completed after 0 minute(s) and 0 seconds,
[02:00:00] [INF] [27] TaskManager: ExecuteQueuedTasks,
[02:00:00] [INF] [25] TaskManager: Daily trigger for Extract Chapter Images set to fire at 06/12/2020 02:00, which is 23:59:59.0033711 from now.,
[02:44:56] [INF] [6] Jellyfin.Api.Auth.CustomAuthenticationHandler: CustomAuthentication was not authenticated. Failure message: Access token is required.,
[03:02:37] [INF] [6] Jellyfin.Api.Auth.CustomAuthenticationHandler: CustomAuthentication was not authenticated. Failure message: Access token is required.,`

**Comment by PleaseStopAsking** [comment]
I am also running into this issue upon a fresh install but am not using any reverse proxy. The logs report 

```[21:14:39] [INF] [4] Main: Jellyfin version: 10.5.5,
[21:14:39] [INF] [4] Main: Arguments: ["/jellyfin/jellyfin.dll", "--datadir", "/config", "--cachedir", "/cache", "--ffmpeg", "/usr/lib/jellyfin-ffmpeg/ffmpeg"],
[21:14:39] [INF] [4] Main: Operating system: Linux,
[21:14:39] [INF] [4] Main: Architecture: X64,
[21:14:39] [INF] [4] Main: 64-Bit Process: True,
[21:14:39] [INF] [4] Main: User Interactive: True,
[21:14:39] [INF] [4] Main: Processor count: 2,
[21:14:39] [INF] [4] Main: Program data path: /config,
[21:14:39] [INF] [4] Main: Web resources path: /jellyfin/jellyfin-web,
[21:14:39] [INF] [4] Main: Application directory: /jellyfin/,
[21:14:39] [INF] [4] ServerConfigurationManager: Setting cache path: /cache,
[21:14:39] [INF] [4] App: Loading assemblies,
[21:14:40] [WRN] [4] Emby.Server.Implementations.Library.UserManager: No users, creating one with username root,
[21:14:40] [INF] [4] ServerConfigurationManager: Saving system configuration,
[21:14:40] [INF] [4] ServerConfigurationManager: Setting cache path: /cache,
[21:14:42] [INF] [4] Jellyfin.Server.Migrations.MigrationRunner: Marking all known migrations as applied because this is a fresh install,
[21:14:42] [INF] [4] Main: Kestrel listening on all interfaces,
[21:14:42] [WRN] [4] Microsoft.AspNetCore.DataProtection.KeyManagement.XmlKeyManager: No XML encryptor configured. Key {0b15b0dd-be37-4ed0-83f0-425e35e3bf1e} may be persisted to storage in unencrypted form.,
[21:14:42] [INF] [4] App: Running startup tasks,
[21:14:42] [INF] [4] TaskManager: Daily trigger for Extract Chapter Images set to fire at 07/03/2020 02:00, which is 4:45:17.1063935 from now.,
[21:14:42] [INF] [4] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Found ffmpeg version 4.2.1,
[21:14:43] [INF] [4] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Available decoders: ["h264", "h264_cuvid", "hevc", "hevc_cuvid", "mpeg2video", "aac", "ac3", "mp3"],
[21:14:43] [INF] [4] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Available encoders: ["libx264", "h264_amf", "h264_nvenc", "h264_v4l2m2m", "h264_vaapi", "libx265", "hevc_amf", "hevc_nvenc", "hevc_vaapi", "mpeg4", "msmpeg4", "aac", "ac3", "libmp3lame", "libopus", "libvorbis", "srt"],
[21:14:43] [INF] [4] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: FFmpeg: SetByArgument: /usr/lib/jellyfin-ffmpeg/ffmpeg,
[21:14:43] [INF] [4] App: ServerId: dab22dd7a255446a86b62c4371c5d2c4,
[21:14:43] [INF] [4] App: Executed all pre-startup entry points in 0:00:00.393855,
[21:14:43] [INF] [4] App: Core startup complete,
[21:14:44] [INF] [7] Dlna: Registering publisher for urn:schemas-upnp-org:device:MediaServer:1 on 172.23.0.4,
[21:14:44] [INF] [4] App: Executed all post-startup entry points in 0:00:00.9702508,
[21:14:44] [INF] [4] Main: Startup complete 0:00:05.2407555,
[21:14:45] [INF] [4] TaskManager: StartupTrigger fired for task: Update Plugins,
[21:14:45] [INF] [4] TaskManager: Queueing task PluginUpdateTask,
[21:14:45] [INF] [5] TaskManager: Executing Update Plugins,
[21:14:46] [INF] [4] TaskManager: Update Plugins Completed after 0 minute(s) and 1 seconds,
[21:14:47] [INF] [4] TaskManager: ExecuteQueuedTasks,
[21:14:52] [INF] [7] Jellyfin.Api.Auth.CustomAuthenticationHandler: CustomAuthentication was not authenticated. Failure message: Access token is required.,
[21:14:54] [INF] [13] Jellyfin.Api.Auth.CustomAuthenticationHandler: CustomAuthentication was not authenticated. Failure message: Access token is required.,
[21:14:54] [INF] [4] Jellyfin.Api.Auth.CustomAuthenticationHandler: CustomAuthentication was not authenticated. Failure message: Access token is required.,
[21:14:54] [INF] [4] ServerConfigurationManager: Saving system configuration,
[21:14:54] [INF] [4] ServerConfigurationManager: Setting cache path: /cache,
[21:14:54] [INF] [7] Jellyfin.Api.Auth.CustomAuthenticationHandler: CustomAuthentication was not authenticated. Failure message: Access token is required.,
[21:15:03] [INF] [13] Jellyfin.Api.Auth.CustomAuthenticationHandler: CustomAuthentication was not authenticated. Failure message: Access token is required.,
[21:15:04] [INF] [12] Jellyfin.Api.Auth.CustomAuthenticationHandler: CustomAuthentication was not authenticated. Failure message: Access token is required.,
[21:15:05] [INF] [7] Jellyfin.Api.Auth.CustomAuthenticationHandler: CustomAuthentication was not authenticated. Failure message: Access token is required.,
[21:15:05] [INF] [7] Jellyfin.Api.Auth.CustomAuthenticationHandler: CustomAuthentication was not authenticated. Failure message: Access token is required.,
[21:15:05] [INF] [7] ServerConfigurationManager: Saving system configuration,
[21:15:05] [INF] [7] ServerConfigurationManager: Setting cache path: /cache,
[21:15:08] [INF] [12] Jellyfin.Api.Auth.CustomAuthenticationHandler: CustomAuthentication was not authenticated. Failure message: Access token is required.,
[21:15:08] [INF] [12] ServerConfigurationManager: Saving system configuration,
[21:15:08] [INF] [12] ServerConfigurationManager: Setting cache path: /cache,
[21:15:10] [INF] [15] Jellyfin.Api.Auth.CustomAuthenticationHandler: CustomAuthentication was not authenticated. Failure message: Access token is required.,
[21:15:10] [INF] [15] ServerConfigurationManager: Saving system configuration,
[21:15:10] [INF] [15] ServerConfigurationManager: Setting cache path: /cache,
[21:15:10] [INF] [7] Jellyfin.Api.Auth.CustomAuthenticationHandler: CustomAuthentication was not authenticated. Failure message: Access token is required.,
[21:15:10] [INF] [7] ServerConfigurationManager: Saving system configuration,
[21:15:10] [INF] [7] ServerConfigurationManager: Setting cache path: /cache,
[21:15:10] [INF] [15] Jellyfin.Api.Auth.CustomAuthenticationHandler: CustomAuthentication was not authenticated. Failure message: Access token is required.,
[21:15:10] [INF] [15] ServerConfigurationManager: Saving system configuration,
[21:15:10] [INF] [15] ServerConfigurationManager: Setting cache path: /cache,
[21:15:50] [INF] [18] Jellyfin.Api.Auth.CustomAuthenticationHandler: CustomAuthentication was not authenticated. Failure message: Access token is required.,
[21:15:50] [INF] [18] ServerConfigurationManager: Saving system configuration,
[21:15:50] [INF] [18] ServerConfigurationManager: Setting cache path: /cache,
[21:15:51] [INF] [19] Jellyfin.Api.Auth.CustomAuthenticationHandler: CustomAuthentication was not authenticated. Failure message: Access token is required.,
[21:15:51] [INF] [19] ServerConfigurationManager: Saving system configuration,
[21:15:51] [INF] [19] ServerConfigurationManager: Setting cache path: /cache,
[21:15:51] [INF] [19] Jellyfin.Api.Auth.CustomAuthenticationHandler: CustomAuthentication was not authenticated. Failure message: Access token is required.,
[21:15:51] [INF] [19] ServerConfigurationManager: Saving system configuration,
[21:15:51] [INF] [19] ServerConfigurationManager: Setting cache path: /cache,
```

I can walk through the initial setup screens but when I get to the 'Configure Remote Access' page, it proceeds no further.

I just had this up and running a week ago with the same exact docker-compose file but now its failing.

```
  jellyfin:
    image: linuxserver/jellyfin
    container_name: jellyfin
    hostname: jellyfin
    restart: unless-stopped
    ports:
      - 8096:8096
      - 8920:8920
    volumes:
      - ~/docker_configs/jellyfin/config:/config
      - ~/docker_configs/jellyfin/cache:/cache
      - /srv:/media
    environment:
      - TZ=US/Eastern
```

**Comment by radriaanse** [comment]
Issue seems to be fixed here with 10.6, I did however not change anything in the new network setting that came out of the box.
Maybe that the changes in #2774 had effect on this?
